MDANSE_GUI/pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools", "numpy", "pip", "wheel"]
+requires = ["setuptools", "numpy", "pip", "wheel", "netCDF4"]
```

Comment:
Was it necessary to bring back netCDF4? It has been one of our objectives to drop it altogether and use only HDF5.

Reply:
I added it back in as there were some import errors. Its this being used in MDANSE_GUI\PyQtGUI\pygenplot\models\data_tree_model.py.

```(test) PS C:\Users\xcb63893\PycharmProjects\MDANSE\MDANSE_GUI\dist> mdanse_gui.exe                                 
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\xcb63893\AppData\Local\anaconda3\envs\test\Lib\site-packages\MDANSE_GUI\Scripts\mdanse_gui.py", line 20, in main
    from MDANSE_GUI.PyQtGUI.main import startGUI
  File "C:\Users\xcb63893\AppData\Local\anaconda3\envs\test\Lib\site-packages\MDANSE_GUI\PyQtGUI\main.py", line 21, in <module>
    from MDANSE_GUI.PyQtGUI.MainWindow import Main
  File "C:\Users\xcb63893\AppData\Local\anaconda3\envs\test\Lib\site-packages\MDANSE_GUI\PyQtGUI\MainWindow.py", line 69, in <module>
    from MDANSE_GUI.PyQtGUI.pygenplot.widgets.main_window import MainWindow
  File "C:\Users\xcb63893\AppData\Local\anaconda3\envs\test\Lib\site-packages\MDANSE_GUI\PyQtGUI\pygenplot\widgets\main_window.py", line 20, in <module>
    from MDANSE_GUI.PyQtGUI.pygenplot.models.data_tree_model import (
  File "C:\Users\xcb63893\AppData\Local\anaconda3\envs\test\Lib\site-packages\MDANSE_GUI\PyQtGUI\pygenplot\models\data_tree_model.py", line 20, in <module>
    import netCDF4
ModuleNotFoundError: No module named 'netCDF4'
```